Wrote command-line script to use the library.

ypid opened the pull request.

ypid committed:
Wrote command-line script to use the library.

schmijos commented:
Cool one. Thx!

Would you mind changing the file extension into `.py` or is there a special reason for that you left it away?

Also could please conform to at least [PEP8](https://pypi.python.org/pypi/pep8)?

schmijos commented on html_table_to_csv_converter:
```diff
@@ -0,0 +1,137 @@
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```
Would you mind to remove this license header? You can instead change the LICENSE of the whole project into AGPL if you want.

schmijos commented on html_table_to_csv_converter:
```diff
@@ -0,0 +1,137 @@
+                tables[index]
+            )
+    else:
+        pprint(table)
```
This should be `pprint(tables)`.

ypid committed:
Removed license header from script and changed license of the project to AGPLv3.

* Suggested by @schmijos
* https://github.com/schmijos/html-table-parser-python3/pull/2#discussion_r36173812:
> Would you mind to remove this license header? You can instead change
> the LICENSE of the whole project into AGPL if you want.

ypid commented:
Fixed. Thanks. There is a reason why the script is not called `html_table_to_csv_converter.py` and that is because it is a command line script and it is uncommon to execute programs which are in your `$PATH` with *.py …

ypid committed:
Fixed typo and PEP8 compliance.

ypid committed:
Optimized convert_td_to_float function.
